spec/models/miq_region_spec.rb

```diff
@@ -55,11 +55,5 @@
       MiqRegion.seed
       expect(MiqRegion.first.region).to eq(@region_number)
     end
-
-    it "raises Exception if db region_id doesn't match my_region_number" do
-      @db = FactoryGirl.create(:miq_database)
-      allow(MiqRegion).to receive_messages(:my_region_number => @region_number + 1)
-      expect { MiqRegion.seed }.to raise_error(Exception)
-    end
```

Comment:
This wasn't really testing anything.
The test is not properly stubbing the region number.

So this test actually properly detects the region number (1) and not `@region_number` so it was failing.
I opted to just put in the `miq_database_spec.rb` test that ensures the region of 1 is detected.

Despite being so simple, it is using real detection and is the best region test we have.